go.mod

```diff
@@ -0,0 +1,3 @@
+module tflitego
```

Comment:
As this repository isn't not in the standard library, a name with a FQDN should be provided, what about: `github.com/nbortolotti/tflitego`?

Reply:
definitely @marcoshuck